lib/ansible/modules/cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -392,6 +402,15 @@ def ensure_routes(vpc_conn, route_table, route_specs, propagating_vgw_ids,
                 if e.error_code == 'DryRunOperation':
                     pass
 
+        for route_spec in route_specs_to_recreate:
```

Comment:
Above this line can you please do:
`if not check_mode:`

Reply:
Addressed.